code/src/Survey.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import Summary from './Summary';
+
+// all states
+export const Survey = () => { 
+const [showSummary, setShowSummary] = useState(false)
+const [location, setLocation] = useState("")
+const [radioButton1, setCandyCanes] = useState("")
+const [radioButton2, setGingerBreadHouses] = useState("")
+const [radioButton3, setChristmasChocolates] = useState("")
+const [feedback1, setPositiveFeedback] = useState("")
+const [feedback2, setImproveFeedback] = useState("")
+
+// display summary when the questions has been answered and submitted
+const handleSubmit = (event) => {
+    setShowSummary(true)
+}
+
+// what to display in the summary
+if (showSummary === true) {
+    return (
+        <Summary 
+        location={location}
+        radioButton1={radioButton1}
+        radioButton2={radioButton2}
+        radioButton3={radioButton3}
+        feedback1={feedback1}
+        feedback2={feedback2}
+        />
+    )
+}
+
+return (
+    <>
```

Comment:
You don't actually need to use React Fragment here because everything is already wrapped in the `<form>` tag 😄

Reply:
Is it not that I have to wrap everything in <> since there are more than one HTML element? Because if I delete it, everything turns red ⚠️ 🙈

Reply:
Ahh ok,  I saw wrong 😆